rpa-builder/6.2/modules/ROOT/pages/toolbox-aws-document-processing.adoc

```diff
@@ -20,6 +20,9 @@ The ERP Docs with AWS enables you to use Amazon Textract to analyze invoices and
 The Key values with AWS action step enables you to extract form data by sending a local file to the AWS Textract service, which returns the extracted key-value pairs.
 * xref:toolbox-aws-operations-ocr-with-aws.adoc[OCR With AWS]
 +
+The Queries with AWS Action Step enables you to extract specific data from a document by specifying a query parameter so that the response contains the data you need, without having to apply further filtering on the JSON Output to get the relevant values.
```

Comment:
is "Action Step" a proper noun?

Reply:
Is "Output" a proper noun?

If these are in the UI, should they be bold? If not, lowercase?